onboarding_project/squad_pantry_app/models.py

```diff
@@ -19,31 +24,59 @@ class Dish(models.Model):
         validators=[MinValueValidator(1)], help_text='Time Taken to Prepare the Dish')
 
     def __str__(self):
-        return self.dish_name
+        return self.dish_name+" ("+str(self.prep_time_in_minutes)+" Min)"
 
 
 class Order(models.Model):
+    CANCEL_SUCCESS = 100
+    WRONG_USER = -100
+    PROCESSING_ORDER = 200
+    ORDER_CLOSED_ERROR = -200
+
     ORDER_PLACED = 0
-    REJECTED = 1
-    ACCEPTED = 2
+    ACCEPTED = 1
+    REJECTED = 2
     CANCELLED = 3
     PROCESSING = 4
-    DELIVERY = 5
+    DELIVERED = 5
+
+    CLOSED_ORDERS = [REJECTED, CANCELLED, DELIVERED]
+
     STATUS = (
         (ORDER_PLACED, 'Order Placed'),
         (REJECTED, 'Rejected'),
         (ACCEPTED, 'Accepted'),
         (CANCELLED, 'Cancelled'),
         (PROCESSING, 'Processing'),
-        (DELIVERY, 'Delivered')
+        (DELIVERED, 'Delivered')
     )
+    placed_by = models.ForeignKey(SquadUser, on_delete=models.CASCADE)
     status = models.IntegerField(choices=STATUS, default=ORDER_PLACED)
     dish = models.ManyToManyField(Dish, through='OrderDishRelation')
     scheduled_time = models.DateTimeField(
         blank=True, null=True,
         help_text='Schedule Your Order. Leave it blank for getting your order as soon as possible')
     created_at = models.DateTimeField(auto_now_add=True)
-    closed_at = models.DateTimeField(blank=True, null=True)
+    closed_at = models.DateTimeField(blank=True, null=True, editable=False)
+
+    def save(self, force_insert=False, force_update=False, using=None, update_fields=None):
```

Comment:
Since you are not modifying `force_insert` and other arguments, you can get away with writing:
```py
def save(self, *args, **kwargs):
    <logic>
    return super(Order, self).save(*args, **kwargs)
```
What does `*args` and `**kwargs` mean in python?

Reply:
Done!
They're basically used to handle `variable` number of `arguments` and `key-value` pairs respectively.